cypress/support/commands.ts

```diff
@@ -0,0 +1,208 @@
+import { LoginPagePo } from '@/cypress/e2e/po/pages/login-page.po';
```

Comment:
This file has moved to `cypress/support/commands/commands.ts`, where it contains generic commands. Anything that makes api requests now lives in `cypress/support/commands/rancher-api-commands.ts`

Reply:
i'll move the `logout` command to `cypress/support/commands/commands.ts` as well since it doesn't make an api request, it only intercepts. https://github.com/rancher/dashboard/blob/b35d004126fc2b1a9a0972b55e858d692a15f83b/cypress/support/commands/rancher-api-commands.ts#L63-L70

Reply:
The `cypress/support/commands.ts` file still exists. Was there anything besides logout added there, if so it can be deleted?

Reply:
Yes it can be deleted. Nothing else was added there.